src/supervisor/input_parser.star

```diff
@@ -58,16 +59,21 @@ def _parse_instance(supervisor_args, supervisor_name, superchains, registry):
     # in the parsed config, but this is a tradeoff that we are willing to make
     supervisor_params["superchain"] = superchain
 
+    # Rust or Go supervisor implementation, i.e. kona-supervisor or op-supervisor.
+    # Default is op-supervisor (Go).
+    supervisor_type = supervisor_params["type"]
```

Comment:
Here pls follow a pattern from `/src/participant/cl/input_parser.star` using the `_IMAGE_IDS` and `_default_image` function - it also serves as input validation